mkt/developers/models.py

```diff
@@ -193,41 +191,6 @@ class Meta:
         db_table = 'user_inapp_keys'
 
 
-class PreloadTestPlan(ModelBase):
-    addon = models.ForeignKey('webapps.Webapp')
```

Comment:
Maybe you can drop the constraint - just switch the field to an `IntegerField` and hopefully the migration should do it for you ? Needs to be tested.